src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -170,37 +170,6 @@ def erwerbsm_rente_vor_grundr_m(
     return out
 
 
-@dates_active(end="2020-12-31")
-def erwerbsm_rente_m(erwerbsm_rente_vor_grundr_m: float) -> float:
-    return erwerbsm_rente_vor_grundr_m
-
-
-@dates_active(start="2021-01-01", change_name="erwerbsm_rente_m")
```

Comment:
fiy @TeBackh, we decided to exclude Grundrente from this PR for now as the required changes will be more involved. @nafetsk @MaxBlesch will likely work on it in a new PR in the future.

Reply:
ok